ui/src/app/cohort-review/participant-status/participant-status.component.ts

```diff
@@ -50,7 +56,11 @@ export class ParticipantStatusComponent implements OnInit, OnDestroy {
 
   ngOnInit() {
     this.subscription = this.state.participant$
-      .subscribe(participant => this.participant = participant);
+      .subscribe(participant => {
+        this.participant = participant;
+        this.statusControl.setValue(this.participant
```

Comment:
NIT: I'd prefer this to be one line unless it breaks the 100-char rule, particularly since we now have a line which is half a ternary operator and half another argument entirely.  If you think inlining it is too much for one line for readability I'd suggest breaking it out into its own assignment, e.g.

```
const val = this.participant ? this.participant.status : '';
this.statusControl.setValue(val, /* etc */);
```

Reply:
Done.